### DISCUSSION TITLE
GitHub Octernship - Time Commitment

### DISCUSSION THREAD

**Discussion by bonzonic** [community/community#49696] [GitHub Education] [answered]
Can we undergo a GitHub Octernship while studying at a University? What would be the time commitment per week?

**Comment by samus-aran** [comment, marked as answer]
Hi @bonzonic 👋 

Awesome you're here. 

The GitHub Octernship program has not set any explicit time-commitments as this depends on the nature of the project , their objectives and of course your academic responsibilities. 🧑‍🎓 You can assess how much time you can dedicate by reviewing the projects available before submitting your interest. 👍 

There are mentorships available but this depends on the partnership and their projects. As you are able to browser the available projects and select which ones interests you most. The projects can vary in duration from one to six months.

If you have more questions then please [create a discussion ](https://github.com/orgs/community/discussions/categories/github-education?discussions_q=is%3Aopen+SayedAlAhlMohamed+category%3A%22GitHub+Education%22)and the Education Community team will be able to assist ⭐

**Reply by bonzonic** [reply]
Thanks for clarifying @samus-aran! I'll make sure to read through the projects before applying :)
